pkg/topology/test/helper_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package test_test
+
+import (
+	"encoding/binary"
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/bee/pkg/topology/test"
+)
+
+func TestRandomAddressAt(t *testing.T) {
+	base := swarm.MustParseHexAddress("ca1e9f3938cc1425c6061b96ad9eb93e134dfe8734ad490164ef20af9d1cf59c")
+
+	bitsInCommon := 16
```

Comment:
Perhaps it would be cooler with something not stopping at byte boundary

Reply:
yeah i actually wanted to have this test do all the test cases from 0 to over a byte (or over 16 if to be exact). will do the change